### PR TITLE
[scripts] [combat-trainer] [crossing-trainer] [magic-training][mm] switching from DRCA > DRCMM

### PR DESCRIPTION
**Pull request by MahtraDR**
Tests well locally. Fingers crossed!

### CODE REVIEW

**Comment by MahtraDR** [comment]
When this one gets merged...

resolves https://github.com/rpherbig/dr-scripts/issues/4216

Closes https://github.com/rpherbig/dr-scripts/pull/5207
Closes https://github.com/rpherbig/dr-scripts/pull/5281

File: combat-trainer.lic [hunk 1]
```diff
@@ -1855,7 +1855,7 @@ class SpellProcess
       @equipment_manager.stow_weapon(game_state.weapon_name)
     end
 
-    data = update_astral_data(data)
+    data = DRCMM.update_astral_data(data,@settings)
```
Comment (KatoakDR) [inline combat-trainer.lic:1858]: 🧹adding space between args 
```suggestion
    data = DRCMM.update_astral_data(data, @settings)
```

**Review by KatoakDR — COMMENTED** [review]
Looks good to me. This is waiting on https://github.com/rpherbig/dr-scripts/pull/5543

**Comment by rpherbig** [comment]
Why not also fix [magic-training.lic](https://github.com/rpherbig/dr-scripts/blob/5841a24e0c2cff6200a9df3a6f6bf42acf11aca2/magic-training.lic) in this PR?

**Comment by rpherbig** [comment]
> This is waiting on https://github.com/rpherbig/dr-scripts/pull/5543

@KatoakDR thank you for always being meticulous re: which PRs are dependent. It helps a _lot_ when reviewing and merging.

**Comment by KatoakDR** [comment]
> Why not also fix [magic-training.lic](https://github.com/rpherbig/dr-scripts/blob/5841a24e0c2cff6200a9df3a6f6bf42acf11aca2/magic-training.lic) in this PR?

Good catch! @MahtraDR the relevant line is https://github.com/rpherbig/dr-scripts/blob/5841a24e0c2cff6200a9df3a6f6bf42acf11aca2/magic-training.lic#L70

**Comment by MahtraDR** [comment]
> > This is waiting on https://github.com/rpherbig/dr-scripts/pull/5543
> 
> 
> 
> @KatoakDR thank you for always being meticulous re: which PRs are dependent. It helps a _lot_ when reviewing and merging.

Sorry about that. It's my first time doing PRs like this and I done goofed. I'll do it the Katoak way next time.

**Comment by rpherbig** [comment]
Looks good to me ✔️

**Comment by MahtraDR** [comment]
They're out of order now, but this depends on https://github.com/rpherbig/dr-scripts/pull/5567 Moving it to draft until that's soaked post-merge for a few days.

**Comment by MahtraDR** [comment]
@KatoakDR Would you mind glancing through this? I don't think I've missed any of these, but could do with a double check.

Note I'm purposely not cleaning out DRCA just yet.

**Review by KatoakDR — COMMENTED** [review]
@MahtraDR the changes look good to me. Do you have a moon mage that can cast a ritual spell to test the change in combat-trainer? I don't have any ritual spells

**Comment by MahtraDR** [comment]
> @MahtraDR the changes look good to me. Do you have a moon mage that can cast a ritual spell to test the change in combat-trainer? I don't have any ritual spells

Yes. :) My moonie has been using this version of combat trainer for almost a week now, no issues. He casts IOTS in, and out of combat perfectly, and his telescope always goes to the correct place now.

**Pull request merged**